pkg/db-repo/repository_test.go

```diff
@@ -675,6 +676,111 @@ func TestRepository_DeleteHasManyNoRelation(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+type gormSetTypeValid struct {
+	T    []uint8
+	TSet []string
+}
+
+type gormSetTypeInvalid1 struct {
+	T    []uint8
+	TSet string
+}
+
+type gormSetTypeInvalid2 struct {
+	T    []string
+	TSet []string
+}
+
+type gormSetTypeInvalid3 struct {
+	TSet []string
+}
+
+func TestGormParseMysqlSetType(t *testing.T) {
+	assert.NotPanics(t, func() {
+		scope := gorm.Scope{}
+		db_repo.GormParseMysqlSetType(&scope)
+	}, "should not panic on nil")
+	assert.NotPanics(t, func() {
+		scope := gorm.Scope{
+			Value: &gormSetTypeValid{
+				T:    []uint8{97, 44, 98, 44, 99},
```

Comment:
What happens if T is nil? If T contains something else than utf8?

Reply:
still works :)